_episodes/07-branches.md

```diff
@@ -101,9 +101,16 @@ $ git branch
 ~~~
 {: .output}
 
-Before we used the `checkout` command to checkout a commit using
-commit hashs or `HEAD`. This is similar but now we have given our most
-recent commit in the branch a name, the branch name.
+Before we used the `checkout` command to checkout a file from a specific commit
+using commit hashes or `HEAD` and the filename (`git checkout HEAD <file>`). The
+`checkout` command can also be used to checkout an entire version of the
```

Comment:
I think this section is improved.  I suggest this line say "...to checkout an entire previous version of the..." so it is a little clearer that you are talking about checking out a previous commit.